trackpy/plots.py

```diff
@@ -257,6 +257,56 @@ def pairwise(iterable):
     return ax
 
 
+def annotate3d(centroids, image, **kwargs):
+    """
+    An extension of annotate that distinguishes between 2D, 2D multichannel,
+    3D, and 3D multichannel images, just as in pims.frame.__repr_html_. See
+    annotate for parameters. Returns either (for 2D) an axis object or (for 3D)
+    a scrollable stack.
+    """
+    import matplotlib.pyplot as plt
+    from pims.display import to_rgb, scrollable_stack
+
+    if kwargs.get('ax') is None:
```

Comment:
I left it out because of the `plt.show()` inside `@make_axes`. I don't want the plot to show, I only want to see the HTML object produced by `scrollable_stack`